cmd/fortifyExecuteScan.go

```diff
@@ -61,16 +62,28 @@ func fortifyExecuteScan(config fortifyExecuteScanOptions, telemetryData *telemet
 
 func runFortifyScan(config fortifyExecuteScanOptions, sys fortify.System, command fortifyExecRunner, telemetryData *telemetry.CustomData, influx *fortifyExecuteScanInflux, auditStatus map[string]string) error {
 	log.Entry().Debugf("Running Fortify scan against SSC at %v", config.ServerURL)
-	artifact, err := versioning.GetArtifact(config.BuildTool, config.BuildDescriptorFile, &versioning.Options{}, command)
-	if err != nil {
-		return fmt.Errorf("unable to get artifact from descriptor %v: %w", config.BuildDescriptorFile, err)
-	}
-	gav, err := artifact.GetCoordinates()
-	if err != nil {
-		return fmt.Errorf("unable to get project coordinates from descriptor %v: %w", config.BuildDescriptorFile, err)
+
+	fortifyProjectName := config.ProjectName
+	fortifyProjectVersion := config.PullRequestName
+	if len(fortifyProjectName) == 0 || len(fortifyProjectVersion) == 0 {
```

Comment:
@OliverNocon and I had discussed to avoid such kind of configuration options in future and to force people to modify their build descriptor coordinates on-the-fly if they need something like that. I am not too strongly opinionated on that one but the numerous variations of mods usually leads to higher support demands.

Reply:
I see. So, ultimate source of truth should be build descriptor files.